docs/Configuration.md

```diff
@@ -104,4 +104,4 @@ These options are only useful with React Native projects.
 | `getPolyfills`                  | `({platform: ?string}) => $ReadOnlyArray<string>`                      | An optional list of polyfills to include in the bundle. The list defaults to a set of common polyfills for Number, String, Array, Object... |
 | `postProcessBundleSourcemap`    | `PostProcessBundleSourcemap`                                           | An optional function that can modify the code and source map of the bundle before it is written. Applied once for the entire bundle.        |
 | `getModulesRunBeforeMainModule` | `(entryFilePath: string) => Array<string>`                             | An array of modules to be required before the entry point. It should contain the absolute path of each module.                              |
-| `postProcessModules`            | `(modules: Array<Module>, entryFiles: Array<string>) => Array<Module>` | A function that can change the resulting modules output.                                                                                    |
+| `processModuleFilter`            | `(module: Array<Module>) => boolean` | A function that can change the resulting modules output.                                                                                    |
```

Comment:
Can you change the description to: `A filter function to discard specific modules from the output`?